### PR TITLE
chore(logging): logger name must be written

### PR DESCRIPTION
**Pull request by gonzalad**
- LOGGER_NAME should be written even if
  locationInfo is false.
- refactored daikon-logging unit tests

**What is the problem this Pull Request is trying to solve?**
 
**What is the chosen solution to this problem?**
 
**Link to the JIRA issue**

 
**Please check if the Pull Request fulfills these requirements**
- [x] The PR commit message follows our [guidelines](../CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features, coverage should be over 75% in the new code)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in JIRA), if any, are all linked or available in the Pull Request

 
**[ ] This Pull Request introduces a breaking change**

### CODE REVIEW

File: daikon-logging/logging-event-layout/src/test/java/org/talend/daikon/logging/layout/AbstractLayoutTest.java [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+import org.junit.Before;
+import org.junit.Test;
+import org.talend.daikon.logging.event.field.LayoutFields;
+import org.talend.daikon.logging.event.layout.LayoutUtils;
+
+/**
+ * @author agonzalez
+ */
+public abstract class AbstractLayoutTest {
```
Comment (sgandon) [inline daikon-logging/logging-event-layout/src/test/java/org/talend/daikon/logging/layout/AbstractLayoutTest.java]: nice idea using an Abstract class with unique tests for all loggers.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/287/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/287/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/288/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/288/)

**Comment by gonzalad** [comment]
> Please could you update the README.md (https://github.com/Talend/daikon/blob/master/daikon-logging/logging-event-layout/README.MD) to add the new loggerName field.

done, I moved the existing logger.name to the standard fields in the README doc.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/289/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/289/)

**Pull request merged**